server/test/channelTest.py

```diff
@@ -0,0 +1,28 @@
+from channel import ChannelManager
+from pkg_resources import get_distribution
+
+manager = ChannelManager()

```

Comment:
per our private discussion this file should be rewritten to subclass unittest.TestCase 

see https://github.com/JakeNarkizian/sound-send/blob/master/server/test/StreamSegmenterTest.py